src/crewai/utilities/prompts.py

```diff
@@ -54,10 +54,12 @@ def _build_prompt(
         response_template=None,
     ) -> str:
         """Constructs a prompt string from specified components."""
-        if not system_template and not prompt_template:
+        if not system_template or not prompt_template:
```

Comment:
Would be great enhance some documentation to keep explict that they are dependents on each other